internals/cli/cli.go

```diff
@@ -80,9 +80,9 @@ var commands []*cmdInfo
 // debugCommands holds information about all debug commands.
 var debugCommands []*cmdInfo
 
-// addCommand replaces parser.addCommand() in a way that is compatible with
+// AddCommand replaces parser.addCommand() in a way that is compatible with
 // re-constructing a pristine parser.
-func addCommand(name, shortHelp, longHelp string, builder func() flags.Commander, optDescs map[string]string, argDescs []argDesc) *cmdInfo {
```

Comment:
This function was not made for being exported. It's already not a great API, but we forgive as it's an internal detail that we can easily fix.  The adoption on Termus is still somewhat of a private use, but still, if we change the shape of this function we'll break every copy of it that is out there. We should probably turn it into a struct with nicely named fields (unlike the parameters) and docs.